docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
 Decred uses the [BLAKE-256](../research/blake-256-hash-function.md) hashing function. PoW mining can be done using GPUs. However, ASICs that mine DCR are now available, and since their introduction the PoW difficulty has become high enough to make GPU
 mining unlikely to be profitable.
 
----
+## Purchasing specialized hardware (ASIC)
 
-## Solo Mining or Pool Mining  
+ASIC are the machines used to mine the most popular cryptocurrencies and can only be used for mining a single algorithm. 
 
-##### <img class="dcr-icon" src="/img/dcr-icons/Solo.svg" /> Solo Mining
+The following Decred ASIC miners are available on the market today:
 
-:fontawesome-solid-exclamation-triangle: **Solo mining is not recommended and is not covered by this documentation!** The Decred network regularly sees a network hash rate of up to 10,000Gh/s. Solo mining is generally only done by advanced individuals or organized groups with a large cluster of GPUs so it is not addressed here.
 
-##### <img class="dcr-icon" src="/img/dcr-icons/Pool.svg" /> Pool Mining
+<table>
```

Comment:
Tables can be formatted with Markdown syntax, raw HTML is not necessary. Here's an example:

https://github.com/decred/dcrdocs/blame/master/docs/research/block-production-times.md#L11-L24

Reply:
Updated